tox.ini

```diff
@@ -1,25 +1,32 @@
 [tox]
-envlist = py36, py37, py38, py39, py310
+envlist = py{36, 37, 38, 39, 310}
+
+[gh-actions]
+python = 
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310
 
 [testenv]
+passenv = GITHUB_*
 deps = 
-    pytest
-    pytest-cov
-    ipython
-    nbsmoke
-
+    -r{toxinidir}/requirements_dev.txt
+usedevelop = true
 commands =
     pytest {posargs}
 
 [pytest]
 testpaths = 
     skrf
-addopts = "--cov=skrf"
+    doc/source/examples
+    doc/source/tutorials
+addopts = --cov
```

Comment:
I don't understand why the command `pytest` test only the `skrf` directory despite the doc is also indicated, but it works... ! ;) could you explain to me this magic ?

Reply:
I'm not hundred percent sure, I think `.ipynb` files aren't even recognized without setting the option `--nbval`.

Reply:
This is the regex pytest is using by default.
[pytest ini parser](https://github.com/pytest-dev/pytest/blob/062d91ab474881b58ae1ff49b4844402677faf45/src/_pytest/python.py#L108)

Reply:
OK. So the test of the notebook is performed via tox because of the `nbval` option.